code-formatter/Dockerfile

```diff
@@ -19,6 +21,8 @@ RUN gem install standardrb
 
 RUN apk add git
 
+RUN git config --global --add safe.directory GITHUB_WORKSPACE
```

Comment:
Is this going to be a literal - does it need to be `$GITHUB_WORKSPACE`?

Reply:
it refers to the arg at the top of the page

Reply:
Sure, but won't the `RUN` argument will see that as the string literal of `GITHUB_WORKSPACE`? I believe for it to output the variable, you'll need to prefix with `$` else it will literally output GITHUB_WORKSPACE (i.e. not what the variable is)?

Reply:
sure will change it